src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -12,164 +12,95 @@
             [status-im2.contexts.communities.overview.style :as style]
             [status-im2.contexts.communities.menus.community-options.view :as options]
             [status-im2.contexts.communities.menus.request-to-join.view :as join-menu]
+            [status-im2.contexts.communities.overview.utils :as utils]
+            [cljs.core :as core]
```

Comment:
`[cljs.core :as core]` could you elaborate?

Reply:
leftover, thanks!